editor/editor_inspector.cpp

```diff
@@ -218,11 +222,16 @@ void EditorProperty::_notification(int p_what) {
 			}
 			check_rect = Rect2(ofs, ((size.height - checkbox->get_height()) / 2), checkbox->get_width(), checkbox->get_height());
 			draw_texture(checkbox, check_rect.position, color);
-			ofs += get_constant("hseparator", "Tree");
-			ofs += checkbox->get_width();
+			ofs += checkbox->get_width() + get_constant("hseparator", "Tree");
 		} else {
 			check_rect = Rect2();
 		}
+		if (EDITOR_DEF("interface/editor/show_type_icons", false)) {
+			if (!type_icon.is_null())
+				draw_texture(type_icon, Vector2(ofs, (size.height / 2) - (type_icon->get_height() / 2)));
+
+			ofs += 20; // Hardcoded value, so there's still a space if there's no icon for the property.
```

Comment:
How does it work with HiDPI and e.g. 200% upscaled icons?

Reply:
Yep, I'll multiply that by EDSCALE right now.